lib/components/src/ScrollArea/OverlayScrollbars.tsx

```diff
@@ -14,17 +14,17 @@ interface OverlayScrollbarsComponentProps extends React.HTMLAttributes<HTMLDivEl
  * https://github.com/KingSora/OverlayScrollbars/pull/218
  * */
 
-export const OverlayScrollbarsComponent: React.FC<OverlayScrollbarsComponentProps> = ({
+export const OverlayScrollbarsComponent: FC<OverlayScrollbarsComponentProps> = ({
   options = {},
   extensions,
   className,
   children,
   ...rest
 }) => {
-  const osTargetRef = React.useRef<HTMLDivElement>();
-  const osInstance = React.useRef<OverlayScrollbars>();
+  const osTargetRef = useRef<HTMLDivElement>();
```

Comment:
Importing as named import can result in the outputted type definitions to be cleaner, we should do this everywhere across the codebase.